### DISCUSSION TITLE
Burgermenu anchor scroll

### DISCUSSION THREAD

**Discussion by SebGbs** [bootscore/bootscore#475] [Q&A]
Hello!
I'm having a problem with the anchor scroll when used in the burgermenu. It doesn't scroll all the way to the anchor but only scrolls a bit down. When I check https://bootscore.me/theme/scrollspy-onepager/ on mobile/tablet I can see the same problem. Using the anchorpoints outside of the burgermenu works just fine.

Does anyone else have this problem and maybe fixed it?

Thanks in advance!

**Comment by crftwrk** [comment]
Hi,

are you on Chrome or Edge? If so, this is a known issue. Check https://github.com/twbs/bootstrap/issues/36681 and https://github.com/bootscore/bootscore/issues/227.

**Reply by SebGbs** [reply]
Ah yes, I've tested it in Chrome. I've read the other posts and increased the transition time on offcanvas.

Thanks for the help :)
